pkg/control_room/control_center.go

```diff
@@ -0,0 +1,92 @@
+package control_room
+
+import (
+	"container/heap"
+
+	"github.com/TNSEngineerEdition/WailsClient/pkg/city"
+	"github.com/umahmood/haversine"
+)
+
+type GraphNode = city.GraphNode
+
+type ControlCenter struct {
+	city         *city.City
+	cachedRoutes map[[2]uint64][]GraphNode
```

Comment:
I'll be better if you use pointers for the graph nodes, it'll be more performant as you don't have to copy everything.
```suggestion
	cachedRoutes map[[2]uint64][]*GraphNode
```